src/common/utils/index.js

```diff
@@ -94,19 +94,22 @@ export const landingPageSteps = [
     color: 'peach',
     number: 1,
     description: 'Enter your email and phone details along with a list of each district you would like to get updates for',
-    Icon: PicLeftOutlined
+    Icon: PicLeftOutlined,
+    iconsrc: 'info'
   },
   {
     color: 'green',
     number: 2,
-    description: 'Check and verify your email to activate your subscription',
-    Icon: MailOutlined
+    description: 'Receive notifications as soon as vaccine slots are updated!',
+    Icon: MailOutlined,
+    iconsrc:  '../../assets/images/notif.svg'
```

Comment:
toh aise nahi hoga how you're importing in that file similarly you'll have to import the asset and pass it on like how MailOutlined is passed